src/github/copilotPrWatcher.ts

```diff
@@ -139,7 +140,11 @@ export class CopilotPRWatcher extends Disposable {
 	private _initialize() {
 		this._getStateChanges();
 		this._pollForChanges();
-		this._register(this._reposManager.onDidChangeAnyPullRequests(() => this._getStateChanges()));
+		this._register(this._reposManager.onDidChangeAnyPullRequests(e => {
+			if (e.some(pr => COPILOT_ACCOUNTS[pr.model.author.login])) {
+				debounce(this._getStateChanges, 50);
```

Comment:
When a PR description is opened, we get several PR changed events. Debounce so we don't refrsh the states many times. Only refresh the state changes if one of the changed PRs was Padawan.